modal/runner.py

```diff
@@ -296,12 +295,8 @@ async def _run_app(
 
     output_mgr = _get_output_manager()
     if interactive and output_mgr is None:
-        warnings.warn(
-            "Interactive mode is disabled because no output manager is active. "
-            "Use 'with modal.enable_output():' to enable interactive mode and see logs.",
-            stacklevel=2,
-        )
-        interactive = False
+        msg = "Interactive mode requires the context manager: `with modal.enable_output():`"
```

Comment:
Mentioned on Slack but I think this error message (and the previous waring) could mislead people who are using the CLI (`modal run --interactive --quiet`). Unfortunately I don't think we can tell (in any obvious way) which way Modal is being invoked here.